### PR TITLE
Added default sleep value to monitoring-integration sync

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
tendrl-bug-id: Tendrl/monitoring-integration#377

Signed-off-by: GowthamShanmugam <gshanmug@redhat.com>

### CODE REVIEW

File: tendrl/monitoring_integration/sync/__init__.py [hunk 1]
```diff
@@ -44,6 +47,8 @@ def run(self):
                         )
                         raise ex
                 except etcd.EtcdKeyNotFound as ex:
+                    # Before cluster import sync_interval is not populated
+                    time.sleep(DEFAULT_SLEEP)
```
Comment (GowthamShanmugam) [inline tendrl/monitoring_integration/sync/__init__.py]: monitoring integration uses sync_intervel from gluster NS, but before import cluster namespace is not available so it consumes lot of cpu

**Comment by GowthamShanmugam** [comment]
@shtripat @r0h4n please review

File: tendrl/monitoring_integration/sync/__init__.py (hunk 1 above)
Comment (shtripat) [reply tendrl/monitoring_integration/sync/__init__.py]: Consumes lot of course as there is no sleep at all while sync loop and actually thread in entering a infinite while loop without any sleep. So it's good to default for sleep if cluster not imported yet.

**Comment by GowthamShanmugam** [comment]
@shtripat It is separate PR only, I tested with 120 sec but it takes too long to update, so I reduced to 60

**Comment by GowthamShanmugam** [comment]
@r0h4n need to merge this

**Pull request merged**